wg-api/spec-documents/scroll-view.md

```diff
@@ -0,0 +1,265 @@
+*`enabled` - The scrollbar will be visible if the contents are larger than the viewport.
+
+#### `scrollView.setVerticalScrollBarMode(mode)` _Experimental_
+
+* `mode` string - Can be `disabled`, `enabled-but-hidden`, `enabled`. Default is `enabled`.
+
+Controls how the vertical scroll bar appears and functions.
+
+#### `scrollView.getHorizontalScrollBarMode()` _Experimental_
+
+Returns `string` - horizontal scrollbar mode.
+
+#### `scrollView.getVerticalScrollBarMode()` _Experimental_
+
+Returns `string` - vertical scrollbar mode.
+
+#### `scrollView.setHorizontalScrollElasticity(elasticity)` _Experimental_
+
+* `elasticity` string - Can be `automatic`, `none`, `allowed`. Default is `automatic`.
+
+The scroll view’s horizontal scrolling elasticity mode.
+A scroll view can scroll its contents past its bounds to achieve an elastic effect. 
+When set to `automatic`, scrolling the horizontal axis beyond its document
+bounds only occurs if the document width is greater than the view width,
+or the vertical scroller is hidden and the horizontal scroller is visible.
+* `automatic` - Automatically determine whether to allow elasticity on this axis.
+* `none` - Disallow scrolling beyond document bounds on this axis.
+* `allowed` - Allow content to be scrolled past its bounds on this axis in an elastic fashion.
+
+#### `scrollView.setVerticalScrollElasticity(elasticity)` _Experimental_
+
+* `elasticity` string - Can be `automatic`, `none`, `allowed`. Default is `automatic`.
+
+The scroll view’s vertical scrolling elasticity mode.
+
+#### `scrollView.getHorizontalScrollElasticity()` _Experimental_
+
+Returns `string` - The scroll view’s horizontal scrolling elasticity mode.
+
+#### `scrollView.getVerticalScrollElasticity()` _Experimental_
+
+Returns `string` - The scroll view’s vertical scrolling elasticity mode.
+
+#### `view.setScrollPosition(point)` _Experimental_
+
+* `point` - The point in the `contentView` to scroll to.
+
+Scroll to the horizontal (`point.x`) and vertical (`point.y`) position.
+
+#### `view.getScrollPosition()` _Experimental_
+
+Returns `Point` - The horizontal and vertical scroll position.
+
+#### `view.getMaximumScrollPosition()` _Experimental_
+
+Returns `Point` - The maximum horizontal and vertical scroll position.
+
+#### Events
+
+#### Event: 'did-scroll' _Experimental_
+
+Returns:
+
+* `event` Event
+
+Emitted when the content view is being scrolled.
+
+#### Event: 'start-scroll' _Experimental_
+
+Returns:
+
+* `event` Event
+
+Emitted at the beginning of user-initiated scroll tracking.
+
+#### Event: 'end-scroll' _Experimental_
+
+Returns:
+
+* `event` Event
+
+Emitted at the end of scroll tracking.
+
+## Rollout Plan
+
+- add the documentation to [Views API (Part 1)](views-api-1-content-view.md)
+- Move the implementation from `BrowserView` to `InspectableWebContentsView` (in particular the implementation of draggable regions)
```

Comment:
I'd like to learn more about `InspectableWebContentsView`, haven't seen it mentioned elsewhere in the RFC 😮